app/busy_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package app
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBusySet(t *testing.T) {
+	busy := &Busy{}
+
+	require.False(t, busy.IsBusy())
+
+	busy.Set(time.Second * 30)
+	require.True(t, busy.IsBusy())
+
+	expire := busy.Expires()
+	require.Greater(t, expire.Unix(), time.Now().Add(time.Second*10).Unix())
+
+	busy.Clear()
+	require.False(t, busy.IsBusy())
```

Comment:
Maybe add a case where you set the busy value when is already set (I know is under control, but would be another test case to add).

Reply:
Added additional test cases.